src/rsdl.cpp

```diff
@@ -20,7 +20,7 @@ Event::EventType Event::get_type() const {
       return LCLICK;
     if (e.type == SDL_MOUSEBUTTONDOWN && e.button.button == SDL_BUTTON_RIGHT)
       return RCLICK;
-    if (e.type == SDL_KEYDOWN)
```

Comment:
Why? 🤔 and… should not we do the same for `SDL_KEYUP`?

Reply:
Key down and key press are two different concepts. A key press happens only once when the user presses a key. Key down is an event that occurs for the duration that the user holds the key down.
SDL Events have additional info to differentiate these, despite having the same event type. 

While SDL sends Key Down event for every frame that the key is down, the Key Up event is only sent once. So the same change is not needed for key release.

An alternative is to rename the event type to KEY_DOWN, but that takes away the ability to identify the first frame that the key was pressed.

Reply:
[I was down the last night. :))]